src/shared/components/results/ResultsData.tsx

```diff
@@ -32,11 +32,23 @@ const ResultsData: FC<{
   direct?: boolean;
   selectedEntries: string[];
   handleEntrySelection: (id: string) => void;
-}> = ({ resultsDataObject, direct, selectedEntries, handleEntrySelection }) => {
-  const [namespace] = useNS();
+  namespaceOverride?: Namespace;
+  displayIdMappingColumns?: boolean;
+}> = ({
+  resultsDataObject,
+  direct,
+  selectedEntries,
+  handleEntrySelection,
+  namespaceOverride,
+  displayIdMappingColumns,
+}) => {
+  const namespace = useNS() || namespaceOverride || Namespace.uniprotkb;
```

Comment:
Naming-wise, now that I see it in context, I'd call that `namespaceFallback` then, as it's not really overriding the internal value so behaviour might be unexpected